### PR TITLE
chore: adding instance creation to network service tiers docs

### PR DESCRIPTION
**Pull request by mark1000**
Destined for
https://cloud.google.com/network-tiers/docs/using-network-service-tiers#creating_vm_instances

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_terraform
 Failed Terraform check. More details below. 
 ```bash 
Running terraform fmt
examples/network_service_tiers/main.tf
--- old/examples/network_service_tiers/main.tf
+++ new/examples/network_service_tiers/main.tf
@@ -53,7 +53,7 @@
     }
   }
   network_interface {
-    network    = "default"
+    network = "default"
     access_config {
       network_tier = "STANDARD"
     }
Error: terraform fmt failed with exit code 3
Check the output for diffs and correct using terraform fmt <dir>
 ```

**Comment by morgante** [comment]
@mark1000 Looks like this has merge conflicts, can you PTAL?

**Comment by mark1000** [comment]
I think I fixed the merge issue. PTAL? Thanks!

**Comment by mark1000** [comment]
@morgante Do you know why the int-trigger failed? Is there a way to retry it?

File: examples/network_service_tiers/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# This file is maintained automatically by "terraform init".
```
Comment (morgante) [inline examples/network_service_tiers/.terraform.lock.hcl:1]: This file should not be committed.

**Review by morgante — CHANGES_REQUESTED** [review]
Test failure is unrelated.

**Comment by mark1000** [comment]
Deleted. Thanks.

**Review by morgante — APPROVED** [review]

**Pull request merged**